workflows/VGP-assembly-v2/Scaffolding-HiC-VGP8/Scaffolding-HiC-VGP8-tests.yml

```diff
@@ -0,0 +1,49 @@
+- doc: Test outline for Scaffolding-HiC-VGP8
+  job:
+    input GFA:
+      class: File
+      location: https://www.dropbox.com/scl/fi/slbz2m742u01wf0d4x0ra/input-GFA.gfa1?rlkey=uyj3p0ngw9wyumhsja55go2rh&dl=1
+      filetype: gfa1
+    HiC Forward reads:
+      class: File
+      location: https://www.dropbox.com/scl/fi/disaq6nirslhlnzc7fvzn/HiC-Forward-reads.fastqsanger.gz?rlkey=p2nf0zskp2bwk6fy0qtournzn&dl=1
+      filetype: fastqsanger.gz
+    HiC reverse reads:
+      class: File
+      location: https://www.dropbox.com/scl/fi/4nh43txcan0e7wp3rqb8w/HiC-reverse-reads.fastqsanger.gz?rlkey=3jr0cuxaged96gxs41j65vzn6&dl=1
+      filetype: fastqsanger.gz
+    Estimated genome size - Parameter File:
```

Comment:
Is there a reason for this not being an integer instead of a file ?

Reply:
The integer is extracted in a file in another workflow, so we keep it in a file to easily make the connection